contrib/python/api/skydive/wsshell.py

```diff
@@ -141,7 +142,11 @@ def main():
                       mode=args.mode,
                       file=file)
     client.connect()
+    client.start()
 
 
 if __name__ == '__main__':
+    logging.basicConfig()
```

Comment:
I think this should be in the main function otherwise this is not going to be called by the skydive-ws-client(entry_points)

Reply:
This should be called only once per process, thus in the main. I can add a comment in the code that initialization of the logging module should be called for the logs to work.

Reply:
If you believe I should do otherwise, I am open to suggestions :)

Reply:
my point was that placing the log initialization here we don't get any output with command line client `skydive-ws-client`

ex: skydive-ws-client listen

moving these lines within the main function directly solves the issue and we get outputs.

Reply:
OK, will fix.

Reply:
Maybe should also consider another patch to support log levels, so we will not have all the debug prints all the time :)